modules/options/src/main/scala/scala/build/actionable/ActionableDiagnostic.scala

```diff
@@ -6,17 +6,6 @@ import scala.build.Position
 import scala.build.errors.Diagnostic.TextEdit
 import scala.build.errors.{Diagnostic, Severity}
 
-abstract class ActionableDiagnostic extends Diagnostic {
-
-  /** Provide the new content which will be replaced by actionable diagnostic
-    */
-  def suggestion: String
-
-  override def severity = Severity.Hint
-
-  override def textEdit: Option[TextEdit] = Some(TextEdit(suggestion))
-}
-
```

Comment:
So the main reason I removed this is that it's only really used internally and not really necessary. Plus before this there was the possibility of having a `TextEdit` in any `Diagnostic` without any `suggestion` meaning that there wouldn't be a title or anything to tie to it. This just removes this and instead like I said in the description make sure there is always a title with the `TextEdit`, which means that just with the `Diagnostic` we have everything we need to create a `ScalaAction`  with.

Reply:
LGTM